src/Elliptic/Systems/Elasticity/Tags.hpp

```diff
@@ -69,8 +67,8 @@ struct Stress : db::SimpleTag {
 };
 
 /*!
- * \brief The energy density \f$U\f$ stored in the deformation of the
- * elastic material.
+ * \brief The energy density \f$U=-\frac{1}{2}S_{ij}T^{ij}\f$ stored in the
```

Comment:
you deleted the `\`

Reply:
I don't know what you mean. I added one; there's `\f$`, `\frac..` is new and `\f$`again.

Reply:
you're right, I didn't read the diff correctly, nvm :)